src/LinuxTracing/PerfEventReaders.cpp

```diff
@@ -303,11 +303,11 @@ SchedWakeupPerfEvent ConsumeSchedWakeupPerfEvent(PerfEventRingBuffer* ring_buffe
       .timestamp = ring_buffer_record.sample_id.time,
       .ordered_stream = PerfEventOrderedStream::FileDescriptor(ring_buffer->GetFileDescriptor()),
       .data =
-          {
-              // The tracepoint format calls the woken tid "data.pid" but it's effectively the
-              // thread id.
-              .woken_tid = sched_wakeup.pid,
-          },
+          {// The tracepoint format calls the woken tid "data.pid" but it's effectively the
+           // thread id.
+           .woken_tid = sched_wakeup.pid,
+           .was_blocked_by_thread = static_cast<pid_t>(ring_buffer_record.sample_id.tid),
+           .was_blocked_by_process = static_cast<pid_t>(ring_buffer_record.sample_id.pid)},
```

Comment:
```suggestion
           .was_blocked_by_process = static_cast<pid_t>(ring_buffer_record.sample_id.pid),},
```

If you add a comma after the last field, clang-format formats in a more readable way, like it was before.